webserver/views/user.py

```diff
@@ -7,6 +7,7 @@
 
 user_bp = Blueprint("user", __name__)
 
+
 @user_bp.route("/lastfmscraper/<user_id>.js")
```

Comment:
Not part of this commit, but we thought that this may not be a good place for this url, as it is in the `/user/` namespace (effectively preventing us having a user called `lastfmscraper`, however rare it may be)